test/sagemaker_tests/huggingface_pytorch/training/integration/local/test_single_machine_training.py

```diff
@@ -59,9 +59,9 @@ def tokenize(batch):
     test_dataset = test_dataset.map(tokenize, batched=True, batch_size=len(test_dataset))
 
     # set format for pytorch
-    train_dataset.rename_column_("label", "labels")
+    train_dataset =  train_dataset.rename_column("label", "labels")
```

Comment:
Is this change backward compatible between datasets versions 1.5 and 1.6?

Reply:
Yes `train_dataset =  train_dataset.rename_column("label", "labels")` was introduced in `1.4.0`